mlf_core/cli.py

```diff
@@ -170,7 +170,7 @@ def sync(project_dir, set_token, pat, username, check_update) -> None:
             sys.exit(1)
         sys.exit(0)
 
-    syncer = TemplateSync(new_template_version='', project_dir=project_dir_path, gh_username=username, token=pat, repo_owner=project_data['github_username'])
+    syncer = TemplateSync(new_template_version='', project_dir=project_dir_path, gh_username=username, token=pat)
```

Comment:
I do not remember this being removed from CT. 
Why can we remove this now?

Reply:
I was wondering the same thing, but we removed it in cookietemple as well. Reason being, the repo owner will be directly determined from the .mlf_core.yml , as there we have all info we need and we parse the data anyway.